### PR TITLE
added back text; added new item to secret config

### PR DESCRIPTION
**Pull request by Gkrumbach07**
closes: https://github.com/opendatahub-io/odh-dashboard/issues/1251

## Description

added back text and added a new item to the secret config

## How Has This Been Tested?

go to pipelines page when there are no pipelines os you see the empty view

## Test Impact

n/a

## Request review criteria:

- [x] The commits have meaningful messages (squashes happen on merge by the bot).
- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

### CODE REVIEW

**Review by andrewballantyne — APPROVED** [review]
Looks good. I'll have to test it before EOD, but the code is what I expected.

cc @harshad16 

/hold

File: frontend/src/concepts/pipelines/elyra/utils.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export const generateElyraSecret = (
         engine: 'Tekton',
         auth_type: 'KUBERNETES_SERVICE_ACCOUNT_TOKEN',
         api_endpoint: route,
+        public_api_endpoint: `${route}/pipelineRuns/${namespace}`,
```
Comment (harshad16) [inline frontend/src/concepts/pipelines/elyra/utils.ts:60]: We need the dashboard route here 

for example:
api_endpoint: https:://<ds-pipeline-api-server>
public_api_endpoint: https://<rhods-dashboard>/pipelineRuns/<namespace>

**Review by harshad16 — CHANGES_REQUESTED** [review]
route seems to be wrong.
we want rhods-dashboard host which displays data-science pipelines 
is this reflecting the behaviour ?

File: frontend/src/concepts/pipelines/elyra/utils.ts (hunk 1 above)
Comment (Gkrumbach07) [reply frontend/src/concepts/pipelines/elyra/utils.ts:60]: so would the public api endpoint just be /`pipelineRuns/<namespace>` Im confused on what rhods-dashboard would be?

Comment (andrewballantyne) [reply frontend/src/concepts/pipelines/elyra/utils.ts:60]: ah right -- you wanted the host name not the route name

@Gkrumbach07 can we use the origin of Dashboard + `/pipelineRuns/the-namespace`?

Comment (harshad16) [reply frontend/src/concepts/pipelines/elyra/utils.ts:60]: correct for public_api_endpoint, we want hostname of RHODS Dashboard in place of route.
route would point to ds-pipeline API server

**Review by andrewballantyne — APPROVED** [review]
Okay, verified... looks good now.

/unhold

This is what I got after the server was created.
```json
{
  "display_name": "Data Science Pipeline",
  "metadata": {
    "tags": [],
    "display_name": "Data Science Pipeline",
    "engine": "Tekton",
    "auth_type": "KUBERNETES_SERVICE_ACCOUNT_TOKEN",
    "api_endpoint": "https://ds-pipeline-pipelines-definition-andrews-regular-user-test.apps.not.a.host.redhat.com",
    "public_api_endpoint": "https://odh-dashboard-opendatahub.apps.not.a.host.redhat.com/pipelineRuns/andrews-test-namespace",
    "cos_auth_type": "USER_CREDENTIALS",
    "cos_endpoint": "https://s3.amazonaws.com/",
    "cos_bucket": "...some-bucket...",
    "cos_username": "...some-key...",
    "cos_password": "...some-secret...",
    "runtime_type": "KUBEFLOW_PIPELINES"
  },
  "schema_name": "kfp"
}
```

**Pull request merged**